src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        String randomColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(101);
```

Comment:
make `101` constant

Reply:
what is the point of it?

Reply:
dont start conversations on Github, use Mate chats instead
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants

before submitting PR, you should carefully examine this files in the tasks to avoid such common mistakes